scripts/setup_py_configure.py

```diff
@@ -205,19 +205,22 @@ def get_wheel_dependencies():
     regex_group_depends = \
         r'set\s*\(\s*ITK\_GROUP\_([a-zA-Z0-9\_\-]+)\_DEPENDS\s*([a-zA-Z0-9\_\-\s]*)\s*'  # noqa: E501
     pattern = re.compile(regex_group_depends)
+    sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+    from itkVersion import get_versions
```

Comment:
```
from itkVersion import get_versions  # noqa: F401
```

Or if you anticipate side-effect due to the update of `sys.path`, you could instead do:

```
itk_root_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
itkVersion=dict()
execfile(os.path.join(itk_root_dir, "itkVersion.py", itkVersion)
version = itkVersion['get_versions']()['package-version']
```

Reply:
Thanks for the review :book: 

Checking with `flake8`, I do not get an `F401` warning -- we should be good since `get_versions` is used.

Thanks for the tip on the `sys.path` mangling. Since we are calling this file as a simple script and there is not anything that should interfere in `ITKPythonPackage`, I think we should be ok.

Reply:
> Checking with flake8, I do not get an F401 warning

humm .. but you may seen few other ones. See #108